flow/packet-headers/ipv4.yaml

```diff
@@ -122,6 +122,100 @@ components:
             default: 0.0.0.0
             features: [count, metric_tags]
           x-field-uid: 14
+        options:
+          type: array
+          minItems: 0
+          items:
+            $ref: '#/components/schemas/Flow.Ipv4.Options'
+          x-field-uid: 15
+    Flow.Ipv4.Options:
+      description: >-
+        IPv4 options are optional extensions for the IPv4 header that can be utilised to provide additional information about the IPv4 datagram. 
+        It is encoded as a series of type, length and value attributes. 
+        The IP header length MUST be increased to accommodate the extra bytes needed to encode the IP options.
+        Currently IP options supported are:
+        1. router_alert Option allows devices to intercept packets not addressed to them directly as defined in RFC2113.
+        2. custom option is provided for to be able to configure user defined IP options as needed.
+      type: object
+      properties:
+        choice:
+          type: string
+          default: router_alert
+          x-field-uid: 1
+          x-enum:
+            router_alert:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        custom:
+          $ref: '#/components/schemas/Flow.Ipv4.Custom'
+          x-field-uid: 3
+    Flow.Ipv4.Custom:
```

Comment:
Not sure if the schema is okay to be Flow.Ipv4.Custom  or better to make it  Flow.Ipv4Options.Custom